### DISCUSSION TITLE
custom treeview widget?

### DISCUSSION THREAD

**Discussion by pombears** [TomSchimansky/CustomTkinter#686] [General]
Hello. 
Just want to ask if there are any plans to add a custom treeview widget (more modern look and feel than the standard one that comes with tkinter). 
If not, is it possible to have widgets from the standard tkinter model and widgets from the customtkinter module co-exist in the same 'root' instance?

**Comment by abhaymakes** [comment]
Yes, it is possible for tkinter and customtkinter widgets to co-exist in the same `root`, however the `root` you are using should be of `CTk()` and not `Tk()`.

**Reply by pombears** [reply]
That is good to know. Thank you

**Comment by Akascape** [comment]
@pombears  Here is an example you can see:
https://github.com/TomSchimansky/CustomTkinter/discussions/524#discussioncomment-3898307

**Reply by pombears** [reply]
Thanks very much Akascape. That's exactly what I was looking for.
